src/views/process-panel/information-card.tsx

```diff
@@ -0,0 +1,123 @@
+import { DispatchProp, connect } from 'react-redux';
+import { MoreOptionsIcon, ProcessIcon } from '~/components/icon/icon';
+import { DetailsAttribute } from '~/components/details-attribute/details-attribute';
+import { RootState } from '~/store/store';
+import { ContextMenuKind } from '~/views-components/context-menu/context-menu';
+import { openContextMenu } from '~/store/context-menu/context-menu-actions';
+
+type CssRules = 'card' | 'iconHeader' | 'label' | 'value' | 'chip' | 'headerText' | 'link';
+
+const styles: StyleRulesCallback<CssRules> = (theme: ArvadosTheme) => ({
+    card: {
+        marginBottom: theme.spacing.unit * 2,
+        paddingBottom: theme.spacing.unit * 3,
+        position: 'relative'
+    },
+    iconHeader: {
+        fontSize: '1.875rem',
+        color: theme.customs.colors.green700
+    },
+    label: {
+        display: 'flex',
+        justifyContent: 'flex-end',
+        fontSize: '0.875rem',
+        marginRight: theme.spacing.unit * 3
+    },
+    value: {
+        textTransform: 'none',
+        fontSize: '0.875rem',
+    },
+    link: {
+        fontSize: '0.875rem',
+        color: theme.palette.primary.main,
+        '&:hover': {
+            cursor: 'pointer'
+        }
+    },
+    chip: {
+        height: theme.spacing.unit * 3,
+        width: theme.spacing.unit * 12,
+        backgroundColor: theme.customs.colors.green700,
+        color: theme.palette.common.white,
+        fontSize: '0.875rem',
+        borderRadius: theme.spacing.unit * 0.625,
+        position: 'absolute',
+        top: theme.spacing.unit * 2.5,
+        right: theme.spacing.unit * 8,
+    },
+    headerText: {
+        fontSize: '0.875rem',
+        marginLeft: theme.spacing.unit * 3,
+    }
+});
+
+interface ProcessInformationCardDataProps {
+    item: ProcessResource;
+}
+
+type ProcessInformationCardProps = ProcessInformationCardDataProps & DispatchProp & WithStyles<CssRules>;
+
+export const ProcessInformationCard = withStyles(styles)(
+    connect((state: RootState) => ({
+        item: state.collectionPanel.item
+    }))(
+        class extends React.Component<ProcessInformationCardProps> {
+            render() {
+                const { classes } = this.props;
+
+                return <div>
+                    <Card className={classes.card}>
+                        <CardHeader
+                            avatar={<ProcessIcon className={classes.iconHeader} />}
+                            action={
+                                <IconButton
+                                    aria-label="More options"
+                                    onClick={this.handleContextMenu}>
+                                    <MoreOptionsIcon />
+                                </IconButton>}
+                            title="Pipeline template that generates a config file from a template"
+                            subheader="(no description)" />
+                            <Chip label="Complete" className={classes.chip} />
```

Comment:
Fix styles for this chip.

Reply:
@kowalczykpawel7 I checked the component and title covers sometimes general status. That must by fixed (you can use text ellipsis) . Also please try to avoid `absolute` styles. They're ok but not in this case

Reply:
![error](https://user-images.githubusercontent.com/13416778/44773155-20379300-ab70-11e8-9b19-59208fccdad1.png)